Add forecasting plot in scenario 1 (#92)
ChrisRackauckas/ASKEM_Evaluation_Staging: docs/src/Scenario1/Evaluation_Scenario_1.md

plt_a1 = plot(sol, leg = :topright)
```

We showcase a probabilistic forecast of the number of infected over the three different age groups, young (`I_A1`), middle (`I_A2`) and old (`I_A3`). `LogNormal` priors are used for the elements of the contact matrix and `rec_disease` parameter. We use 50 samples from the prior distribution to generate the forecast for all cases of contact matrix variations below.

```@example scenario1
param_priors = create_priors(prob, fill(1 / 3, (3, 3)), 0.001)
plot_uncertainty_forecast(prob, [variable( Symbol("I_A1(t)")), variable( Symbol("I_A2(t)")), variable( Symbol("I_A3(t)"))], 0.0:1:100.0, param_priors, 50, seriescolor = [:red :green :blue])
```

> Instead of the entire samples, we can also plot the quantiles of the forecast and compare the effect of different values of variance on parameters. This is also repeated for all different contact matrix variations below.

Author: ```suggestion
Instead of the entire samples, we can also plot the quantiles of the forecast and compare the effect of different values of variance in the priors. This is also repeated for all different contact matrix variations below.
```
